cryptoki/src/object.rs

```diff
@@ -396,6 +399,7 @@ impl TryFrom<CK_ATTRIBUTE_TYPE> for AttributeType {
             CKA_VERIFY_RECOVER => Ok(AttributeType::VerifyRecover),
             CKA_WRAP => Ok(AttributeType::Wrap),
             CKA_WRAP_WITH_TRUSTED => Ok(AttributeType::WrapWithTrusted),
+            0x8000_0000..=0xffff_ffff => Ok(AttributeType::VendorDefined(attribute_type)),
```

Comment:
Can you use `CKA_VENDOR_DEFINED` instead of the first value here and could we find a Rust idiomatic way of getting the best value of the `CK_ULONG` type? Could be that on a different architecture it's bigger than `0xffff_ffff` :nerd_face: